templates/cw721-template/src/contract.rs

```diff
@@ -0,0 +1,66 @@
+// Version info for migration
+pub const CONTRACT_NAME: &str = "crates.io:{{project-name}}";
+pub const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+#[cfg(not(feature = "library"))]
+pub mod entry {
```

Comment:
Doesn't need to be a module inside of a module. : )

See: https://github.com/CosmosContracts/code-and-chill/blob/main/contracts/cw721-piggy-bank/src/contract.rs

Don't need the entry module here.